gradle/libs.versions.toml

```diff
@@ -1,23 +1,23 @@
 [versions]
-kotlinVersion = "1.6.10"
+kotlinVersion = "1.7.20"
 
 # android
 androidAppCompatVersion = "1.2.0"
-googleServicesVersion = "4.3.8"
-firebaseCoreVersion = "17.4.3"
-firebaseCrashlyticsVersion = "17.1.0"
+googleServicesVersion = "4.3.14"
+firebaseAnalyticsVersion = "21.1.1"
+firebaseCrashlyticsVersion = "18.2.3"
 
 # kotlin multiplatform
-napierVersion = "2.4.0"
+napierVersion = "2.6.1"
 
 # moko
-mokoCrashReportingVersion = "0.3.0"
+mokoCrashReportingVersion = "0.3.1"
 
 
 [libraries]
 # android
 appCompat = { module = "androidx.appcompat:appcompat", version.ref = "androidAppCompatVersion" }
-firebaseCore = { module = "com.google.firebase:firebase-core", version.ref = "firebaseCoreVersion" }
+firebaseAnalytics = { module = "com.google.firebase:firebase-analytics", version.ref = "firebaseAnalyticsVersion" }
```

Comment:
why we should use analytics instead of core?

Reply:
https://firebase.google.com/support/release-notes/android

Release from 12 October, include recommendation for migrate on firebase-analytics from firebase-core. I thought replace used libraries on ktx versions, but im don't know will compatibility be maintained.
If u think what my solution include mistake with this library, i should replace on core.